chrome/content/addFinger.js

```diff
@@ -0,0 +1,36 @@
+var { interfaces: Ci, utils: Cu, classes: Cc } = Components;
+
+Cu.import("resource:///modules/imServices.jsm");
+Cu.import("resource:///modules/imXPCOMUtils.jsm");
+Cu.import("chrome://otr/content/otr.js");
+
+XPCOMUtils.defineLazyGetter(this, "_", () =>
+  l10nHelper("chrome://otr/locale/addfinger.properties")
+);
+
+var args = window.arguments[0].wrappedJSObject;
+
+var otrAddFinger = {
+  onload: function() {
+    document.title = _("addfinger.title", args.buddy);
+  },
+  
+  oninput: function(e) {
+    document.documentElement.getButton("accept").disabled = (e.value.length != 40);
```

Comment:
This should probably ensure you're getting hex chars, no?  What about spacing?  People might have be pasting in `11111111 22222222 etc.`

Also, the window just says "Fingerprint".  I think we can be more descriptive. As in, "If you know the 40 hex char fingerprint of your contact's OTR private key, enter it now."

Maybe the "Cancel" button should be "Skip" instead so it's clear that it isn't required.